### PR TITLE
TRIVIAL: gooddata-dbt workflows integration

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/_version.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # (C) 2023 GoodData Corporation
-# Based on
-# https://packaging.python.org/en/latest/guides/single-sourcing-package-version/
+from importlib import metadata
 
-__version__ = "1.4.0"
+try:
+    __version__ = metadata.version("gooddata-sdk")
```
Comment (jaceksan) [inline gooddata-dbt/gooddata_dbt/_version.py:5]: Really "gooddata-sdk"? Should it be "gooddata-dbt"?

**Review by jaceksan — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/_version.py (hunk 1 above)
Comment (hkad98) [reply gooddata-dbt/gooddata_dbt/_version.py]: Thanks for the catch.

**Review by jaceksan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR367

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**